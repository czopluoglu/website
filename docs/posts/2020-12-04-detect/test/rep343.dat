11010000101000011011
00101110101010001100
11101111111001100110
01101000101110100110
01100101000001001110
11101111111100111111
00000000010001011010
01001101011010000000
10011000100011001101
10111111110010000000
00000000101110101001
10110011011010001011
01001111000110110111
01101100101010001010
00001001100111110111
10100111110100001111
01000001001110100010
00101000001010011101
10001000100101100010
10101101101110111010
11000001001100101111
00101010001011111011
11011000100011110110
11010101001111111111
11000100100101001100
11001101011000110110
11101110011000000111
01111011111010000011
00000101001100100110
01000000011100111111
11111101101111111011
01110110100010101111
10001101001010001011
01010110001110111111
11110001111110000100
11011111111011111011
11110000000110101011
10001011001001100110
10111111100011101110
01101100010000001011
00001100001100010111
10001001011011101110
10001001111000101001
11110000001000111110
10110011111011011011
00100001011111000001
10100001011110111110
00100101001111011101
11011001111011101101
10000000001100100101
10001101101100001100
01011000010000101010
10001001101100110111
10001001010010111111
11000111001100111111
11100001010011111110
00001001001110001110
10000000110000000000
11110111111110100010
01100100011111110110
01000000001100011111
00001111110100010011
10010101110111100111
11111111111011111111
00010000011011010110
00000000011011001011
10011001101101111111
00010000011011100011
11000001010001001110
11111001111101010111
00000001001000100100
10001001100110001100
10000101101001010010
11111011110010111010
11001011111110000010
10101001011111111111
00001001101111111111
00000101001001110110
11011001010110111010
01001110011011001111
11101111001111110111
00010101110011100101
10100111101111101000
11101101100101011011
10100101010101101110
11000001001100001010
10010100111110001101
10010101101100111111
01000101010010100001
11100001111010101111
00001001101010000111
00000111011010001110
00000101001010100011
11101001110110110010
10001000111010100110
01101010111111111111
11111100111010000111
01110000001101111111
00000001110010001011
11110101111100000110
01010001010001011100
10001110110011000110
01001000001111100000
00000010010100100011
11111011011010101111
00001000011010000110
00101111011101111110
00001100100000000000
11011000011101110111
10010000001010011101
11100111111100101111
10111011111111110110
11100101001010110000
00110111110111110111
01101111001101001111
01110110101001000111
11110011111100111111
01100000100110000011
00010000111010110000
01101101110111101111
11011101101000001110
00100100011110100110
10001000001010100100
11110001100010111010
10110110011011111110
11011111011111111110
10000101010011100011
00111111010000101111
10010100011111101111
11001001001011101111
00000100011001111110
10110001111101011111
10110001011011001101
00001001011100100111
10010001111010110110
00110000001001011001
11010001100000011010
00001000010101011110
00010011111010111101
10010001111110100010
10010000001100110001
10101111100111110011
11101101110000010110
11010101110001101001
11011010010000010100
00000001000011001000
01001001111110111001
10000000010000100100
10111010100110001111
01010111011101111111
10000100111111110111
00010011001011000011
00100011111000110111
00101010011111101110
10000001001010110000
10001111110110111111
10011110111001100111
10110111111110111110
10001111111111111101
10100001001000110011
11110111011010101110
10001101111011101011
10111101011001101111
11000101010110000001
00010110011010000011
00001000001010110100
10110100001010001010
01101000111000001110
10111000101110111101
00001100101000110110
00000101001000110101
11111101001111110010
10000001110001010001
10101101110111111110
00000000000100100110
11101100111111110111
10001101001001001010
10111001101111011100
10110001011111010000
00001001100100101010
10010001001000110111
11111111110011001000
11101001010111101010
10001001011010001110
01000101000011100101
01001101010100000111
01010010000110100110
00110111001010001101
10010000001000000011
01011010001110100110
11010010011111110011
00000000011110100110
00010011100000010111
01100111011011101110
00001011110110100110
11010011100010000100
00101011101110001111
00111001101100001010
00000110001000100000
00011110011101111110
11110101111111111111
00000000001001001110
10011011111000101011
11100101111111111111
11100011011111101111
01010101111000001010
11100101001011100100
11000001001111111111
00000001100000000011
11101111111111000110
11000101011010111110
01000101001010101111
01100001111110101010
10010011110111111111
10000111101000000010
10010111111000111011
11011111011101001100
10001001110010000001
00100100001000000010
01110100111100000011
10111001010000001111
11000101111111111111
00011010011001101011
00100001101100110111
10100100000011110111
00111111110110010010
00000001001110100111
10000100101000100000
00000111011000001100
11111101010110111010
11011010011110001011
01000001101110110111
11111101111111111100
11101001100000100111
00110111010111110101
10111001101010010110
01011100000001110011
10001000001100110010
00001101001101101010
01110100011010001110
11111111110000001001
01001100111000110011
00100111011010100110
10111101111000000011
01000100000011100001
01100010011000101110
01011100110110010110
00101100000110110010
00010000101111011111
00000001000110010110
