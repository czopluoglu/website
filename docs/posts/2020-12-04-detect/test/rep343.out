-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep343.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.31
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  5
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.8159
                   IDN index value:   0.7158
                           Ratio r:   0.6514

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    2    2    1    2    2    1    1    2
    3    3    4    5    3    3    5    3    3    5

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    5    8    9

   -----------CLUSTER  2-------------

    2    3    4    6    7   10

   -----------CLUSTER  3-------------

   11   12   15   16   18   19

   -----------CLUSTER  4-------------

   13

   -----------CLUSTER  5-------------

   14   17   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d++++++--+--------+-
   +d-+----+--+---+----
   +-d+---+-+------+-+-
   +++d-++-+--+-------+
   +---d+++-+-++--+--+-
   +--++d+++-++---++-+-
   +--+++d-++---+----++
   --+-++-d++-------+--
   -+-+-+++d++---++---+
   +-+-+-+++d-----+++--
   -----+--+-d-+-++--++
   -+-+++-----d+++++---
   ----+-----++d+++++-+
   ------+----++d++-+-+
   --------+-++++d+-++-
   -+--++--+++++++d----
   --+--+---+-++---d+--
   -------+-+--+++-+d++
   +-+-+++---+---+--+d+
   ---+--+-+-+-++---++d
 
 No cross validation for this DETECT run
